components/director/internal/graphql/tenant.go

```diff
@@ -21,7 +22,7 @@ func (y *Tenant) UnmarshalGQL(v interface{}) error {
 }
 
 func (y Tenant) MarshalGQL(w io.Writer) {
-	_, err := w.Write([]byte(y))
+	_, err := io.WriteString(w, strconv.Quote(string(y)))
```

Comment:
why you don't print error? Additionally, maybe we should decide which logger to use?

Reply:
The error is now printed. As discussed, we will use `logrus`, but in new task, to not block our colleges.